src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -44,13 +44,13 @@ public static void LeaveDebugMode()
         [CLSCompliant(false)]
         public static Process Start(string fileName, string userName, SecureString password, string domain)
         {
-            throw new PlatformNotSupportedException(SR.ProcessStartIdentityNotSupported);
+            throw new PlatformNotSupportedException(string.Format(SR.ProcessStartFeaturesNotSupported, "Password and Domain"));
```

Comment:
this won't get localized correctly in the future, please put placeholder for two variables in the resource string (or split it into two if it is being reused)

Reply:
OK, as I said, I'll do it later (maybe today).